internal/cmd/gocdk/testdata/resource.ct

```diff
@@ -44,28 +44,154 @@ $ gocdk resource add dev blob/foo --> FAIL
 gocdk: Adding "blob/foo" to "dev"...
 Error: resource add: "blob/foo" is not a supported type; use 'gocdk resource list' to see available types
 
-# Provision types that don't need any stdin.
-# TODO(rvangent): Add the other types once they work without input.
+$ echof stdin
+$ gocdk resource add dev blob/azureblob < stdin
+gocdk: Adding "blob/azureblob" to "dev"...
+
+Please enter an Azure location.
+  See https://azure.microsoft.com/en-us/global-infrastructure/locations for more information.
+  default: westus
+Enter a value, or "cancel": gocdk:   added a Terraform provider "azurerm" to "main.tf"
+gocdk:   added a Terraform provider "random" to "main.tf"
+gocdk:   added an output variable "BLOB_BUCKET_URL" to "outputs.tf"
+gocdk:   added an output variable "AZURE_STORAGE_ACCOUNT" to "outputs.tf"
+gocdk:   added an output variable "AZURE_STORAGE_KEY" to "outputs.tf"
+gocdk:   added a new file "azureblob.tf"
+gocdk: Success!
+
 $ gocdk resource add dev blob/fileblob
 gocdk: Adding "blob/fileblob" to "dev"...
 gocdk:   added a Terraform provider "local" to "main.tf"
 gocdk:   added an output variable "BLOB_BUCKET_URL" to "outputs.tf"
 gocdk:   added a new file "fileblob.tf"
 gocdk: Success!
 
+$ echof stdin myprojectid\n
+$ gocdk resource add dev blob/gcsblob < stdin
+gocdk: Adding "blob/gcsblob" to "dev"...
+
+Please enter your Google Cloud project ID (e.g. "example-project-123456")
+Enter a value, or "cancel": 
+Please enter a GCP storage location.
+  See https://cloud.google.com/storage/docs/locations for more information.
+  default: US
+Enter a value, or "cancel": gocdk:   added a Terraform provider "google" to "main.tf"
+gocdk:   added an output variable "BLOB_BUCKET_URL" to "outputs.tf"
+gocdk:   added a new file "gcsblob.tf"
+gocdk: Success!
+
+$ echof stdin
+$ gocdk resource add dev blob/s3blob < stdin
+gocdk: Adding "blob/s3blob" to "dev"...
+
+Please enter an AWS region.
+  See https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html for more information.
+  default: us-west-1
+Enter a value, or "cancel": gocdk:   added a Terraform provider "aws" to "main.tf"
+gocdk:   added an output variable "BLOB_BUCKET_URL" to "outputs.tf"
+gocdk:   added a new file "s3blob.tf"
+gocdk: Success!
+
```

Comment:
Note that many of the resources below would require prompts if done initially (e.g., `docstore/awsdynamodb` requires an AWS region. However, since we've already asked for AWS region earlier (for `blob/s3blob`), the CLI saves that answer and doesn't ask again.